### DISCUSSION TITLE
竖版的集装箱号识别效果挺差

### DISCUSSION THREAD

**Discussion by SUPEROKMAN** [PaddlePaddle/PaddleOCR#12695] [Q&A]
总共训练了500多幅图像，170多张是竖版的集装箱号，标注的时候直接将在一行上的竖形文本框在了一起，而且也是按照顺时针方向从左上为第一点进行标注。不知道这样可否。另外自建了字典，只保留了数字和大写字母。横板的箱号识别的挺好，竖版的就很差。初学，请多多指教。
我用的都是单通道的灰度图进行的训练，这样是不是有问题？
![1-122700001-OCR-LF-C01](https://github.com/PaddlePaddle/PaddleOCR/assets/24601592/18193b6c-0b47-40fa-bc29-81074178d45f)
![1-122720001-OCR-LB-C02](https://github.com/PaddlePaddle/PaddleOCR/assets/24601592/5be570df-3e70-42dc-bdd9-37a889ff62a8)

**Comment by chenhao86** [comment]
官网里面有个AIStudio里面有竖版的一整套教程的。https://aistudio.baidu.com/projectdetail/5766320?channel=0&channelType=0&sUid=5866733&shared=1&ts=1703810647780

**Comment by SUPEROKMAN** [comment]
> 官网里面有个AIStudio里面有竖版的一整套教程的。https://aistudio.baidu.com/projectdetail/5766320?channel=0&channelType=0&sUid=5866733&shared=1&ts=1703810647780

咨询了作者，给了一点建议：
评论内容：1.保持标注一致性  2.独立标注每个数字或字符  可以先试一试把竖列排列的集装箱号单独标注每一个数字或字符使它们在垂直方向上排列,并对结果拼接看看效果。也可以整个框标注 3.简单场景下考虑对图像进行预处理，如旋转文本区域，将其转换为水平方向 4.用支持竖列识别算法模型和标注工具
试过将一竖排分为两个标注，效果改善不多。

**Comment by chenhao86** [comment]
你可以试试不要用自建的字典。生成模型后再试试看

**Comment by wangmengmeng99** [comment]
我觉得你的主要问题是数据集太少了，多爬一些这种集装箱的数据集，或者多实际采集一些数据集。

**Comment by zgsxwsdxg** [comment]
> 总共训练了500多幅图像，170多张是竖版的集装箱号，标注的时候直接将在一行上的竖形文本框在了一起，而且也是按照顺时针方向从左上为第一点进行标注。不知道这样可否。另外自建了字典，只保留了数字和大写字母。横板的箱号识别的挺好，竖版的就很差。初学，请多多指教。 我用的都是单通道的灰度图进行的训练，这样是不是有问题？ ![1-122700001-OCR-LF-C01](https://private-user-images.githubusercontent.com/24601592/293242975-18193b6c-0b47-40fa-bc29-81074178d45f.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODM2NzgyNTksIm5iZiI6MTc4MzY3Nzk1OSwicGF0aCI6Ii8yNDYwMTU5Mi8yOTMyNDI5NzUtMTgxOTNiNmMtMGI0Ny00MGZhLWJjMjktODEwNzQxNzhkNDVmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzEwVDEwMDU1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhZmZmMjdiMGY0NmFhZWM1NGNiNjVmYWNlODNkZWJkZGYwYTE4YmJhMWQxODdjN2VkNzU5Nzk1M2U5MmRlM2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRmpwZWcifQ.eQoGyqU4FYkXcVvRJO3DC9RyTH26ReMDzSnBmPYGgN0) ![1-122720001-OCR-LB-C02](https://private-user-images.githubusercontent.com/24601592/293242995-5be570df-3e70-42dc-bdd9-37a889ff62a8.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODM2NzgyNTksIm5iZiI6MTc4MzY3Nzk1OSwicGF0aCI6Ii8yNDYwMTU5Mi8yOTMyNDI5OTUtNWJlNTcwZGYtM2U3MC00MmRjLWJkZDktMzdhODg5ZmY2MmE4LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzEwVDEwMDU1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThjM2FlN2I2MDM2NzgyYjNlMWNhYzc4YjQ0NzQ0MmUxOWExMTdiZmY4MGQ4MDdmNmUxMWRkYWFhMTQ3MTFjNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRmpwZWcifQ.r9e6-vBEpc2zngdcufGZHesJJImAQgo5olz4tX46jU4)

你好，我也遇到了同样的问题？你解决了吗？如何解决的？
